src/react-components/avatar-editor.js

```diff
@@ -487,16 +516,25 @@ export default class AvatarEditor extends Component {
               <AvatarPreview
                 className="preview"
                 avatarGltfUrl={this.state.previewGltfUrl}
+                onGltfLoaded={this.handleGltfLoaded}
                 {...this.inputFiles}
                 ref={p => (this.preview = p)}
               />
             </div>
             <div className="info">
               <IfFeature name="show_avatar_editor_link">
```

Comment:
Might as well flip this config to `internal` in schema.toml, while you're at it.